### ISSUE TITLE
Issue stuck due QA inactivity

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1884]
Something happened this week: the ticket was merged and closed, and it was at QA phase, but QA was not made. The ticket is https://github.com/yegor256/jpeek/issues/205 . How can `ARC` put the project moving if the `QA` is not reviewing pull requests and issues?

**Comment by g4s8** [comment]
@paulodamaso `ARC` can also say quality verdicts like `QA`: `@0crat quality <good|bad|acceptable>`

**Comment by paulodamaso** [comment]
@g4s8 I didn't know that, it should be explicit in policy. Should `ARC`  just give the quality veredict or he/she should assign the task to himself/herself?

**Comment by g4s8** [comment]
@paulodamaso no, you don't need to assign - `assign` means that assignee will be task performer, it's `DEV` for bugs and `REV` for PRs. You  can just say QA verdict and get 8min QA payment.
Added a line in policy: https://github.com/zerocracy/zerocracy.github.io/pull/66

**Comment by paulodamaso** [comment]
@paulodamaso I think that's good, just put it in scope so I can get my bug points and I'll close it right away
